### ISSUE TITLE
[Bug][A11Y][Password input] Hidden password is clearly read by screen readers

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#488]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

iOS and Android

### App version

0.6.0

### Describe the issue

 Hidden password is clearly read by screen readers

### Expected Behavior

 Hidden password should be read without details as a secured field and just the number of entered characters

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (695)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 689
Related issues: 488
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (695)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 689
Related issues: 488
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed
